File: C/BaekJoon/Bronze/B5_10807.c
#include <stdio.h>

int main() {
  int n;
  scanf("%d", &n);

  int arr[230] = {0};

  for (int i = 0; i < n; i++) {
    int temp;
    scanf("%d", &temp);

    arr[temp + 100] += 1;
  }

  int res;
  scanf("%d", &res);

  printf("%d", arr[res + 100]);

  return 0;
}

File: Java/BaekJoon/Bronze/B5_10807.java
package BaekJoon.Bronze;

import java.util.Scanner;

public class B5_10807 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();

        int[] nums = new int[230];

        for (int i = 0; i < n; i++) {
            int temp = sc.nextInt();

            nums[temp + 100] += 1;
        }

        int res = sc.nextInt();

        System.out.println(nums[res + 100]);

        sc.close();
    }
    
}
